src/netaddress.cpp

```diff
@@ -515,6 +517,29 @@ bool CNetAddr::IsAddrV1Compatible() const
     assert(false);
 }
 
+bool CNetAddr::LimitOutboundByNetgroup() const
+{
+    const auto onlynets = gArgs.GetArgs("-onlynet");
+    if (onlynets.size() != 1) {
+        return true;
```

Comment:
As I mention in my comment [elsewhere](https://github.com/bitcoin/bitcoin/pull/27374#issuecomment-1491935771), I think we can drop this logic.

Reply:
I agree, see below.